src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ClientCertificates.cs

```diff
@@ -48,6 +48,17 @@ public void ClientCertificateOptions_ValueArg_Roundtrips(ClientCertificateOption
             }
         }
 
+        [Theory]
+        [InlineData(ClientCertificateOption.Automatic)]
+        public void ClientCertificates_ClientCertificateOptionsAutomatic_ThrowsException(ClientCertificateOption option)
```

Comment:
This test should be changed to a [Fact] and hardcode the setting of the ClientCertificateOptions to 'Automatic'.